storage/redis/client_test.go

```diff
@@ -220,6 +220,7 @@ func TestCrudValidConnection(t *testing.T) {
 		{
 			"GetKeysList",
 			func(t *testing.T, st storage.KeyValueStore) {
+				t.Skip()
```

Comment:
dun dun dun

Reply:
I'm going to remove. I wanted to make sure nothing else was broken after this test failure.